app/views/admin/budgets/_heading.html.erb

```diff
@@ -8,6 +8,9 @@
   <td class="text-right">
     <%= heading.population %>
   </td>
+  <td class="text-right">
+    <%= heading.allow_custom_content ? t("true_value") : t("false_value") %>
```

Comment:
Please, use complete route for i18n keys, see comment on `admin.yml`

I think it would be better use:

`t("admin.shared.true_value") and t("admin.shared.false_value")`

😉